templates/etc/resolv.conf.epp

```diff
@@ -0,0 +1,53 @@
+<%- |
+Array[Simplib::IP,0,3]     $nameservers    = [],
+Array[Simplib::Domain,0,6] $search         = [],
+Resolv::Sortlist           $sortlist       = [],
+Boolean                    $debug          = false,
```

Comment:
Maintaining the same defaults in both the init.pp and this epp seems problematic.  Is there a reason to have them here?